ztp/ran-site-plan-crd/site-config-crd.yaml

```diff
@@ -0,0 +1,298 @@
+                                will be considered as the value for sites. sites value must match the policies match label for sites.
+                              type: string
+                      clusterNetwork:
+                        description: ClusterNetwork is the list of IP address pools for pods. Default is 10.128.0.0/14 and a host prefix of /23.
+                        type: array
+                        items:
+                          description: |
+                            ClusterNetworkEntry is a single IP address block for
+                            pod IP blocks. IP blocks are allocated with size 2^HostSubnetLength.
+                          type: object
+                          required:
+                            - cidr
+                          properties:
+                            cidr:
+                              description: CIDR is the IP block address pool.
+                              type: string
+                            hostPrefix:
+                              description: |
+                                HostPrefix is the prefix size to allocate to each
+                                node from the CIDR. For example, 24 would allocate
+                                2^8=256 adresses to each node. If this field is
+                                not used by the plugin, it can be left unset.
+                              type: integer
+                              format: int32
+                      machineNetwork:
+                        description: MachineNetwork is the list of IP address pools for machines.
+                        type: array
+                        items:
+                          description: MachineNetworkEntry is a single IP address block for node IP blocks.
+                          type: object
+                          required:
+                            - cidr
+                          properties:
+                            cidr:
+                              description: CIDR is the IP block address pool for machines within the cluster.
+                              type: string
+                      serviceNetwork:
+                        description: |
+                          ServiceNetwork is the list of IP address pools for services default is 172.30.0.0/16.
+                          currently only one entry is supported.
+                        type: array
+                        maxItems: 1
+                        items:
+                          type: string
+                      apiVIP:
+                        description: APIVIP is the virtual IP used to reach the OpenShift cluster's API.
+                        type: string
+                      ingressVIP:
+                        description: IngressVIP is the virtual IP used for cluster ingress traffic.
+                        type: string
+                      manifestsConfig:
```

Comment:
I'm not sure grouping these under "things we need for manifests" is right.  The cpuset info is needed for the workload partitioning, which happens to be a manifest, but it's also going to be needed to populate the day-2 performance profile.

Reply:
we can level it up, just we said apply only day0 config till we figure out how we will make the link with day2 config

Reply:
Agreed.  Maybe just start with a placeholder for performance tuning data, where for now the managementCpuset is the only entry; later we can add other entries?

For now, for SNO anyway, it's safe to assume the managementCpuset is the same in the whole cluster (because there's only ONE node in the cluster anyway.  But I'm curious, @browsell : Is this safe to assume that the performance data could be in the cluster area, or is it more appropriate to have it per-node in the node section?

Reply:
Performance configuration would be mapped to a performance profile which would map to a mcp. So for a standard cluster we can't assume the settings will be cluster wide.  I think we need another section here perhaps to group these by mcp

Reply:
So will apply it per node then ?

Reply:
this is hard to be different per node/MCP for day0 . if that the case needs to be day2 ?

Reply:
That makes sense, actually; a single cluster can have a list of MCP roles, and a list of nodes, and each node refers to one of the MCP roles.

@serngawy : Would you object to introducing a "role" or "pool" concept where we can put the performance data?

Reply:
yes, we will do it but its not possible to have multiple instances of the same  day0 manifest per cluster. its only 1 ISO per cluster. I believe we should ask the AI guys to provide that in the future allowing multiple ISO per cluster to provision different nodes. will set the cpu.set per node for now as we just have 1 node->cluster

Reply:
1 iso per cluster shouldn't matter - The differentiation should be at the manifest-per-role level, and you can inject the same kind of day-0 MC object for the 'master' role and a slightly different one for the 'worker' role, and also 'worker-du' or 'worker-cnf' or whatever in the future.  One ISO should be able to add all of them just fine simultaneously...

Reply:
I noticed that you have cpuset down below; can this whole section be removed now?

Reply:
we agreed to leave it as the chrony manifest required single item and the ntp below is list

Reply:
I thought we agreed below (line 172) that we would remove the single item and use the 1st entry in the list as the item for the chrony manifest?

Reply:
Apparently in a side-conversation, @browsell and @serngawy agreed it's good as-is for now.  I'm cool with that too.